cloudevents/tests/test_http_events.py

```diff
@@ -125,17 +128,18 @@ def test_emit_binary_event(specversion):
 
     # Check response fields
     for key in test_data:
-        assert body[key] == test_data[key]
+        assert body[key] == test_data[key], body
     for key in headers:
         if key != 'Content-Type':
-            assert r.headers[key] == headers[key]
+            attributeKey = key[3:]
+            assert r.headers[attributeKey] == headers[key]
```

Comment:
This was deliberate -- I'm not replying with a CloudEvent, but with HTTP headers with the same names as the CloudEvents attributes. Would you prefer to have the attribute map at the beginning and then convert to HTTP-Binary headers afterwards, rather than stripping the `ce-` prefix here?

I'm willing to do an encode/decode loop through the library, but I'm concerned that in doing so we might end up accidentally producing something which can talk to itself but not to other implementations.